index.js

```diff
@@ -124,6 +125,61 @@ app.post('/', function(req, response) {
 				if (error) throw new Error(error);
 				console.log(body);
 			});
+				
+			var options1 =   {
```

Comment:
Also, can we create one generic function taking parameters as input and returning this options object? Instead of duplicating the same code?